src/client/datascience/notebook/intellisense/pythonKernelCompletionProvider.ts

```diff
@@ -308,9 +308,9 @@ export function filterCompletions(
         return r;
     });
 
-    // If not inside of a string, filter out file names (things with a '.' in them or end with '/')
+    // If not inside of a string, filter out file names (things that end with '/')
     if (!insideString) {
-        result = result.filter((r) => !r.itemText.includes('.') && !r.itemText.endsWith('/'));
+        result = result.filter((r) => !r.itemText.endsWith('/'));
```

Comment:
Updated to remove check for `.`

Reply:
That's likely better, but you'll still get filenames.

Reply:
Like open a cell, run some code, type the first letter of a file in your workspace. It will show up in your completion list

Reply:
Then shouldn't we be excluding items that contain `/` or `\` instead of looking for stuff that just ends with `/`

Reply:
They always contain '/' (if they're multiple levels). Jupyter always uses unix path indicators.

But first level files don't have either of those. They just have a possible '.'

Reply:
Alternatively (instead of trying to filter on . and /) we could do our own file list for the current directory of the kernel. I think that would require us to execute some code though. Might be too slow.

Reply:
Weird I'm not getting any completions for any paths

Reply:
I made your change and this is what I get:

![image](https://user-images.githubusercontent.com/19672699/152058741-71cf8792-f4dc-4158-b8b6-4d2206c54699.png)

Reply:
You can see two files that begin with the letter 'i'

Reply:
Fixed.